apiary_hooks.py

```diff
@@ -0,0 +1,120 @@
+import json
+import os
+
+import django
+import dredd_hooks as hooks
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "ccxcon.settings")
+django.setup()
+
+from courses.models import Course, Module
+
+dummyCourse = dict(
+    uuid="7e0e52d0386411df81ce001b631bdd31",
+    title="Introductory Physics: Classical Mechanics",
+    author_name="David E. Pritchard",
+    description="This is the description",
+    video_url="",
+    overview="This is the overview"
+)
+
+@hooks.before_all
+def make_moudle_price_nullable(transactions):
+    """
+    Dredd doesn't yet expose nullable number fields, so we patch it in the before_all.
+    """
+    for t in transactions:
+        try:
+            schema = json.loads(t['expected']['bodySchema'])
+            schema['properties']['price_per_seat_cents']['type'] = ['number', 'null']
+            t['expected']['bodySchema'] = json.dumps(schema)
+        except KeyError:
```

Comment:
There isn't always going to be a `price_per_seat_cents` field on every json payload we send out. If that field isn't present in the expected response, this would explode. The KeyError traps that and silences it.